api/server/extension_points.go

```diff
@@ -40,31 +40,27 @@ func (s *Server) apiRouteHandlerWrapperFunc(apiHandler fnext.ApiRouteHandler) gi
 	return func(c *gin.Context) {
 		context := c.Request.Context()
 		// get the app
-		appName := c.Param(api.CApp)
-		app, err := s.datastore.GetApp(context, appName)
+		appID := c.MustGet(api.AppID).(string)
+		// get the route TODO
```

Comment:
remove?

it's a little odd that the ordering is get route than app, with ids in place I can't think of any issues though exactly

Reply:
fixed